### DISCUSSION TITLE
可莉好可爱✧⁺⸜(●˙▾˙●)⸝⁺✧

### DISCUSSION THREAD

**Discussion by kongqin1999** [share121/inter-knot#4512] [原神]
![17168909471501c59feaccb2260c99940ba30a25d47109c78eae838a88fba6f66edeb8c733e21 0](https://github.com/user-attachments/assets/ae5ff8c0-0134-4b1a-8382-dabcfac48e06)

**Comment by zytcw** [comment]
٩(๑^o^๑)۶

**Comment by MIMIss-Y** [comment]
宝贝女鹅就是最可爱的

**Comment by nzj2004** [comment]
可莉！可莉！

**Comment by lllliufeng** [comment]
可爱！
